STEERING-COMMITTEE.md

```diff
@@ -46,6 +48,37 @@ the way we run this committee, based on feedback from the community.
 1. Act as the final escalation point and decider for any disputes, issues,
    clarifications, or escalations within the project scope.
 
+## Knative Trademark Committee Charter
+
+1. Manage licensing the Knative brand and decide which things can be called
+   "Knative" and how that mark can be used in relation to other efforts or
+   vendors.
+1. Review and approve conformance rules and tests that define brand use
+   decisions.
+1. Review and approve definition of Knative Core scope.
+1. Only the Trademark Committee can redefine its own charter and authority over
+   Knative Trademark, conformance, and brand decisions.
+
+## Trademark Committee Composition
```

Comment:
Reminder TODO: add clarification around who updates the composition of the trademark committee

Reply:
@ronavn @anicolao @thisisnotapril I'd also love some clarification on who is the decision maker here if the mark were to move to a foundation

Reply:
Ultimately the TM holder has final decision on if/when to transfer the TM. The intention is that the KTC is composed in such a way that their advice on if/when to do that is taken seriously and has influence on that decision.

Reply:
Related: i think we should describe the mechanics of how the KTC is composed and maintained. How do we add/remove org representation? Who gets to participate?

Reply:
Suggestion for text:

"Members of the Trademark Committee are organizations with three or more paid staff contributing to Knative, who between them have 50(?) or more merged PRs for the year, a significant financial investment in Knative infrastructure, promotion, resources, and/or events, and an interest in the success of the Knative project.  Each year, the (KTC/SC?) will review the list of organizations that meet this criteria to determine if any should be added to the Trademark Committee."

Reply:
We decided to use a [15% threshold on CNCF devstats](https://knative.teststats.cncf.io/d/5/companies-table?orgId=1&var-period_name=Last%20year&var-metric=contributions) for now, and the KTC can adjust in the future.